source/dashboards/dashboard-cpu-utilization-details-cores.rst

```diff
@@ -51,19 +50,19 @@ Core Utilization to see if any core is reaching close to 100%.
 
 .. _dashboard-cpu-utilization-details-cores.current:
 
+****************************
 Current CPU Core Utilization
---------------------------------------------------------------------------------
+****************************
 
 The Current CPU Core Utilization metric shows the total utilization of each CPU
 core along with the average utilization of all CPU cores.  Watch for any core
 close to 100% utilization and investigate the root cause.
 
-|view-all-metrics| |this-dashboard|
```

Comment:
This line creates a link to the page's table of contents. Since most of the pages are only a few sections long, this is clutter and navigation over-kill.